dune_api_scripts/store_query_result_for_todays_trading_data.py

```diff
@@ -0,0 +1,29 @@
+from duneanalytics import DuneAnalytics
+import os
+from utils import parse_data_from_dune_query, store_as_json_file
+
+
+# initialize client
+dune = DuneAnalytics(os.environ['DUNE_USER'], os.environ['DUNE_PASSWORD'])
+
+# try to login
+dune.login()
+
+# fetch token
+dune.fetch_auth_token()
```

Comment:
This seems to be needed for every script, maybe something that could go into `duneanalytics` module:
```py
def dune_from_environment():
  """Initialize and authenticate a Dune Analytics client from the current environment."""
  dune = DuneAnalytics(os.environ['DUNE_USER'], os.environ['DUNE_PASSWORD'])
  dune.login()
  dune.fetch_auth_token()
  return dune
```